### ISSUE TITLE
Add support for gpx_file in OSM v0.6 API classes

### ISSUE THREAD

**Issue by HarelM** [OsmSharp/core#33]
I want to update Osm trace using the OSM API and OSM Sharp or to get a trace data.
The following is the response that is returned from an API call:
API address:
https://www.openstreetmap.org/api/0.6/gpx/2463219
Response:
```xml
<osm version="0.6" generator="OpenStreetMap server" copyright="OpenStreetMap and contributors" attribution="http://www.openstreetmap.org/copyright" license="http://opendatacommons.org/licenses/odbl/1-0/">
<gpx_file id="2463219" name="20170726.gpx" lat="42.6896" lon="-84.4321" user="jojkuhiuwer" visibility="identifiable" pending="false" timestamp="2017-07-26T21:04:01Z">
<description>NU3001</description>
<tag>GPSLogger</tag>
</gpx_file>
</osm>
```
I'm not sure, but I don't see that this type of response is defined in the `Osm` class, right?
![image](https://user-images.githubusercontent.com/3269297/28644386-fa3d08da-7261-11e7-976c-b3aad605538e.png)

**Comment by HarelM** [comment]
@xivk, I have created the relevant pull request.
Please let me know if you want me to add anything else.
If not, can you please create another NuGet package with these changes?
Thanks!

**Comment by HarelM** [comment]
P.S. I wasn't sure whether `Visibility` should be an `enum` or not...
If it's not an `enum` the interface is more flexible but also more error prone... your call...

**Comment by HarelM** [comment]
Closed via f83d305
